src/main/java/fr/brouillard/oss/jgitver/cli/RunnableCLI.java

```diff
@@ -53,7 +52,7 @@ static int execute(String[] args, PrintStream normal, PrintStream error) {
         CommandLine cli = new CommandLine(opts);
 
         try {
-            cli.parse(args);
+            cli.parseArgs(args);
```

Comment:
picocli deprecated `parse` and changed the return type in the otherwise equivalent `parseArgs`